source/include/stdbool.readme

```diff
@@ -0,0 +1,31 @@
+
+#ifndef _STDBOOL_H
+#define _STDBOOL_H
+
+/*******************************************************************************
+ * This file contains the definitions specified in stdbool.h. It is provided to
+ * allow coreMQTT to be built using compilers that do not provide their own
+ * stdbool.h defintion.
+ *
+ * To use this file:
+ *
+ *    1) Copy this file into a directory that is in your compiler's include path.
+ *       The directory must be part of the include path for system header files,
+ *       for example passed using gcc's "-I" or "-isystem" options.
+ *
+ *    2) Rename the copied file stdbool.h.
+ *
+ */
+
+#ifndef __cplusplus
+
+/* _Bool was introduced in C99. */
+#define bool    int
+#define false   0
+#define true    1
+
+#endif
+
+#define __bool_true_false_are_defined   1
```

Comment:
What is the significance of this macro?

Reply:
It's part of the C standard to define this macro in stdbool.h along with the other three: https://pubs.opengroup.org/onlinepubs/009695399/basedefs/stdbool.h.html